cypress/support/execute-tests.sh

```diff
@@ -130,7 +130,11 @@ function overrideSettings() {
 }
 
 function startWebServer() {
-  webServerProcess=$(ps -ef | grep http-server | grep -v grep)
+  if [ "$OSTYPE" == "msys" ]; then
+    webServerProcess="$(netstat -ano | findstr 0.0.0.0:8282 | awk '{print $5}')"
```

Comment:
This is less specific than the other command and it is possible that it could kill items that are not `http-server`. Is there a way to do this by the name of the process?

Reply:
Yes it can be done by name of process, but under Windows that is reported as node.exe.  So I'd have to kill all of those. This would help: https://github.com/http-party/http-server/pull/515

Reply:
I updated the code to link to that issue so that this can be changed in the future.